.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,106 @@
+name: Run Tests
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      source:
+        description: 'Source of tests'
+        required: true
+        type: string
+        options:
+          - 'specs'
+          - 'sdk'
+        default: 'sdk'
+      specs_url:
+        description: 'Run tests based on specs'
+        required: false
+        type: string
+        default: ''
+      sdk_version:
+        description: 'Run tests based on SDK'
+        required: false
+        type: string
+        default: 'LATEST'
+      sdk_namespace:
+        description: 'SDK to test'
+        required: true
+        type: string
+        options:
+          - rapid
+        default: 'rapid'
+      endpoint_prefix:
+        description: 'Endpoint to prepend specs paths with'
+        required: true
+        type: string
+      jdk:
+        description: 'JDK version to use'
+        required: true
+        type: string
+        options:
+          - '17'
```

Comment:
Let's include the LTS versions: 8, 11, 17, and 21.

And the same question for the type of this input, should it be a `choice`?